client/components/focusable/index.jsx

```diff
@@ -1,11 +1,15 @@
-/** @format */
 /**
  * External dependencies
  */
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 
+/**
+ * Style dependencies
+ */
+import './style.scss';
```

Comment:
do we also want to drop the import in _components.scss?

Reply:
Yes, good catch! Fixed.